### PR TITLE
Revert "expand on the `getContext()` description"

### PR DESCRIPTION
**Pull request by andre-merzky**
Reverts ExaWorks/job-api-spec#122

See #122

### CODE REVIEW

**Comment by andre-merzky** [comment]
ping @hategan : I hope this is in the gist we discussed: #122 was a mistake of mine,   confusing the roles of context and message.

**Comment by hategan** [comment]
> ping @hategan : I hope this is in the gist we discussed: #122 was a mistake of mine, confusing the roles of context and message.

I think your "mistake" was invited by our discussions not being very clear on one or the other.

File: specification.md [hunk 1]
```diff
@@ -166,7 +162,7 @@ using either a remote or local job management library; application jobs
 are then submitted to the pilot system, which sends them directly to the
 existing pilot job instances for execution, bypassing queuing
 systems/LRMs. The requirements for the APIs used to submit the pilot jobs
-as well as those used to run the application and remote job management
+as well as those used to run the application l and remote job management
```
Comment (hategan) [inline specification.md:165]: l?

**Review by hategan — APPROVED** [review]

File: specification.md [hunk 2]
```diff
@@ -278,7 +274,7 @@ customary in the language in which the library is implemented
 
 #### Interaction with LRMs and Scalability
 
-Implementations should use bulk status operations when interacting with
+Implementations must use bulk status operations when interacting with
```
Comment (hategan) [inline specification.md:277]: Are we sure we want "must" instead of "should" here>

Comment (andre-merzky) [reply specification.md:277]: I am very much in favor of `should` - thus the original edit.  I'll do a separate PR for that one if that's ok for you...

File: specification.md (hunk 1 above)
Comment (andre-merzky) [reply specification.md:165]: Well, the revert also covers some small bug fixes :-/

File: specification.md (hunk 2 above)
Comment (hategan) [reply specification.md:277]: A separate PR sounds like a lot of work. I'm fine with this being kept here.

File: specification.md (hunk 1 above)
Comment (hategan) [reply specification.md:165]: Hah, OK. I think we can safely keep this fix though.

Comment (andre-merzky) [reply specification.md]: ok, done.

File: specification.md [hunk 3]
```diff
@@ -166,7 +162,7 @@ using either a remote or local job management library; application jobs
 are then submitted to the pilot system, which sends them directly to the
 existing pilot job instances for execution, bypassing queuing
 systems/LRMs. The requirements for the APIs used to submit the pilot jobs
-as well as those used to run the application and remote job management
+as well as those used to run the application remote job management
```
Comment (SteVwonder) [inline specification.md:165]: I think we should keep the `and` (I'm assuming this fix was in #122?)

File: specification.md [hunk 4]
```diff
@@ -103,11 +104,6 @@ Resource Managers (LRMs), such as PBS/Torque, SLURM, etc. To a first
 approximation, a job management API is understood as an abstraction layer
 on top of various LRMs.
 
-Job management is sometimes also provided by execution managers, with
-capabilities similar to LRMs but operating in user space, on a limited
-subset of resources such as within a job's allocation.  This job
-management API aims to also transparently abstract such execution managers.
-
```
Comment (SteVwonder) [inline specification.md:110]: I think this paragraph is meaningful and correct.  Do we want to keep it?

**Review by SteVwonder — COMMENTED** [review]
Just some other fixes that were reverted that I think we might want to keep:

File: specification.md (hunk 3 above)
Comment (andre-merzky) [reply specification.md]: Agree and done.

File: specification.md (hunk 4 above)
Comment (andre-merzky) [reply specification.md]: Agree and done.

**Comment by hategan** [comment]
Latest changes look nice. @SteVwonder?

**Comment by SteVwonder** [comment]
Yep! LGTM! Clicking the big green button :)

**Pull request merged**